src/coverlet.core/Reporters/CoberturaReporter.cs

```diff
@@ -133,28 +133,48 @@ public string Report(CoverageResult result)
             return Encoding.UTF8.GetString(stream.ToArray());
         }
 
-        private static IEnumerable<string> GetRootDirs(Modules modules, bool useSourceLink)
+        private static IEnumerable<string> GetBasePaths(Modules modules, bool useSourceLink)
         {
             if (useSourceLink)
             {
                 return new[] { string.Empty };
             }
 
-            return modules.Values.SelectMany(k => k.Keys).Select(Directory.GetDirectoryRoot).Distinct();
+            var pathsGroupedByRootDir = modules.Values.SelectMany(k => k.Keys).GroupBy(Directory.GetDirectoryRoot);
+            var basePaths = pathsGroupedByRootDir.Select(group =>
+            {
+                var splittedPaths = group.Select(x => x.Split(Path.DirectorySeparatorChar)).OrderBy(x => x.Length).ToList();
```

Comment:
rename `x` in `absolutePath` I don't like single letter var name, code is hard to read.

Reply:
Done